build.xml

```diff
@@ -38,6 +38,12 @@
     </fileset>
   </path>
 
+  <path id="xmlunit.classpath">
+    <fileset dir="third_party/xmlunit">
+      <include name="xmlunit-core-2.5.0.jar"/>
```

Comment:
I wonder about the specificity of includes here. Does including the version number buy us anything? The Guava include here (guava/*.jar) seems more natural than the JUnit includes copied from the library. I think I would prefer the Guava style, at least if we're adding new filesets.